tests/test_xpsx_calendar.py

```diff
@@ -0,0 +1,145 @@
+from unittest import TestCase
+
+import pandas as pd
+from pytz import UTC
+from nose_parameterized import parameterized
+
+from .test_trading_calendar import ExchangeCalendarTestBase
+from trading_calendars.trading_calendar import WEEKENDS
+from trading_calendars.exchange_calendar_xpsx import XPSXExchangeCalendar
+
+
+class XPSXCalendarTestCase(ExchangeCalendarTestBase, TestCase):
+
+    answer_key_filename = 'xpsx'
+    calendar_class = XPSXExchangeCalendar
+
+    MAX_SESSION_HOURS = 5.967
+
+    HAVE_EARLY_CLOSES = False
+
+    # Pakistan observes Pakistan Standard Time (PKT, GMT+05:00) all year.
+    DAYLIGHT_SAVINGS_DATES = []
```

Comment:
Instead of needing to overwrite this you can subclass this suite with `NoDSTExchangeCalendarTestBase`, which is defined in `test_trading_calendars.py`.

Reply:
Ah, thanks. I'll make the change for Indonesia and South Africa too.